blockchain/state_manager/src/vm_circ_supply.rs

```diff
@@ -106,13 +106,12 @@ impl GenesisInfoVesting {
     }
 }
 
-fn get_actor_state<DB: Blockstore>(
+fn get_actor_state<DB: Blockstore + Clone>(
```

Comment:
Is this `Clone` used?

Reply:
[yes](https://github.com/ChainSafe/forest/blob/seamless-statetree/utils/forest_shim/src/state_tree.rs#L33)